pkg/controller/sparkapplication/spark_pod_eventhandler.go

```diff
@@ -66,12 +68,13 @@ func (s *sparkPodEventHandler) onPodDeleted(obj interface{}) {
 	if deletedPod == nil {
 		return
 	}
+	glog.V(2).Infof("Pod %s deleted in namespace %s.", deletedPod.GetObjectMeta().GetName(), deletedPod.GetObjectMeta().GetNamespace())
 	s.enqueueSparkAppForUpdate(deletedPod)
 }
 
 func (s *sparkPodEventHandler) enqueueSparkAppForUpdate(pod *apiv1.Pod) {
 	if appKey, ok := createMetaNamespaceKey(pod); ok {
-		glog.V(2).Infof("Enqueuing SparkApplication %s for processing", appKey)
+		glog.V(2).Infof("Enqueuing SparkApplication %s for status update processing.", appKey)
```

Comment:
It's not just for status update. There might be action taken on the app, e.g., restarting it.

Reply:
ok